sections/faqs/browser-support.js

```diff
@@ -0,0 +1,11 @@
+import md from "components/md"
+
+const BrowserSupport = () => md`
+  ## Which browsers does styled-components support?
+
+  Styled-components supports all the browsers that ReactDOM supports.
+  Please do read the [React docs section](https://reactjs.org/docs/react-dom.html#browser-support) to learn more.
+  However, the short version is that ReactDOM currently supports "IE9+", i.e. any browser and version that was released since - and including - IE9. 
```

Comment:
- I'd leave the added prose out here like starting with "however" (especially since I think "however" should follow neither sentence above)
- ~ReactDOM~ React

So something like this would totally be enough:

> All major browsers including Internet Explorer 9 and above are supported.

Reply:
@philpl  Thanks for the thorough review. Very much appreciated. 

So, just to be clear would the following be more suitable for this section?

  ## Which browsers are supported?

  All major browsers including Internet Explorer 9 and above are supported.

Reply:
@andrwsalcdo well, the first sentence should still be included with the modifications that I proposed :wink: The sentence here is only to prevent people from having to go to the React page, while still being vague enough so that it won't need to be changed often

Reply:
@philpl  Okay, maybe my changes are satisfactory. Thanks for giving the necessary guidance and being polite throughout the process. 🥂